plugin.js

```diff
@@ -281,24 +286,64 @@ function processWorklets(t, path, processor) {
   }
 }
 
+const PLUGIN_BLACKLIST_NAMES = [
+  // '@babel/plugin-transform-destructuring', do not blacklist
+  'metro-react-native-babel-preset/node_modules/@babel/plugin-transform-object-assign',
+];
+
+const PLUGIN_BLACKLIST = PLUGIN_BLACKLIST_NAMES.map((pluginName) => {
+  try {
+    const blacklistedPluginObject = require(pluginName);
+    const blacklistedPlugin = blacklistedPluginObject.default({
+      assertVersion: (x) => true,
+    });
+
+    visitors.explode(blacklistedPlugin.visitor);
+    return blacklistedPlugin;
+  } catch (e) {
+    console.log(`No ${pluginName}!`);
```

Comment:
please remove console.log statements. If this is an error let's throw here instead? Or use console.warn or anything that'd suggest you are not accidently commiting some debug/logging code